src/cmd1.cc

```diff
@@ -556,89 +554,6 @@ s16b tot_dam_aux(object_type *o_ptr, int tdam, monster_type *m_ptr,
 
 
 /*
- * Search for hidden things
- */
-void search(void)
```

Comment:
What about hidden doors? Will they just automatically show up, or...?

(I know that I sometimes use the "search" mode and "slug-like" to detect hidden doors in Orc Cave 21 if I have an anti-magic character without detection.)